saleor/payment/utils.py

```diff
@@ -108,7 +108,7 @@ def create_payment(
     if checkout is not None:
         data["checkout"] = checkout
 
-    payment, _ = Payment.objects.get_or_create(defaults=defaults, **data)
+    payment, _ = Payment.objects.get_or_create(gateway=gateway, defaults=defaults, **data)
```

Comment:
`data` is a correct object to store items for `get_or_create`. Could you move the `gateway` there (to keep the consistency)?
Lines 99-104

Reply:
@korycins, I have done the requested change.